### PR TITLE
Update create-sensitivity-labels.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes as the navigation options have changed.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7004

File: microsoft-365/business-video/create-sensitivity-labels.md [hunk 1]
```diff
@@ -31,27 +31,42 @@ Sensitivity labels allow you to classify and protect content that is sensitive t
...
-1. To add a sub-label to a label, select the three dots (more actions), then **Add sub level**.
-1. When finished, choose **Publish labels**, **Choose labels to publish**, and then **Add**. Select the labels you want to publish, and then select **Add**, **Done**, and then **Next**.
-1. By default, the new label policy is applied to everyone. If you want to limit who the policy is applied to, select **Choose users or groups**, and then **Add**. Select who you want the policy to apply to, and then select **Add**, **Done**, and then **Next**.
-1. If you want a default label for documents and email, select the label you want from the drop-down list. Review the remaining settings, adjust as needed, and then select **Next**.
-1. Enter a **Name** and **Description** for your policy. Select **Next**.
-1. Review your settings, then select **Publish**.
+1. From the [Microsoft 365 compliance center](https://compliance.microsoft.com/), select **Solutions** > **Information protection**
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/create-sensitivity-labels.md]:
```suggestion
1. From the [Microsoft 365 compliance center](https://compliance.microsoft.com/), select **Solutions** > **Information protection**.
```

File: microsoft-365/business-video/create-sensitivity-labels.md [hunk 2]
```diff
@@ -31,27 +31,42 @@ Sensitivity labels allow you to classify and protect content that is sensitive t
...
+    
+    If you don't immediately see this option, first select **Show all**.
+
+2. On the **Labels** page, select **+ Create a label** to start the New sensitivity label configuration.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/create-sensitivity-labels.md]:
```suggestion
2. On the **Labels** page, select **+ Create a label** to start the new sensitivity label configuration. 
```

File: microsoft-365/business-video/create-sensitivity-labels.md [hunk 3]
```diff
@@ -31,27 +31,42 @@ Sensitivity labels allow you to classify and protect content that is sensitive t
...
+
+    ![Create a sensitivity label.](../media/create-sensitivity-label-full.png)
+
+3. On the **Define the scope for this label** page, the options selected determine the label's scope for the settings that you can configure and where they will be visible when they are published:
+
+    ![Scopes for sensitivity labels.](../media/sensitivity-labels-scopes.png)
+
+    - If **Files & emails** is selected, you can configure settings that apply to apps that support sensitivity labels, such as Office Word and Outlook. If this option isn't selected, you see the first page of these settings but you can't configure them and the labels won't be available for users to select in these apps.
+
+    - If **Groups & sites** is selected, you can configure settings that apply to Microsoft 365 groups, and sites for Teams and SharePoint. If this option isn't selected, you see the first page of these settings but you can't configure them and the labels won't be available for users to select for groups and site.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/create-sensitivity-labels.md]:
```suggestion
    - If **Groups & sites** is selected, you can configure settings that apply to Microsoft 365 groups, and sites for Teams and SharePoint. If this option isn't selected, you see the first page of these settings but you can't configure them and the labels won't be available for users to select for groups and sites.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Comment by efrene** [comment]
@yogkumgit @msbemba, I am in the process of creating an updated video for this article, so I will update the steps when I post the new video.   Thanks!

**Comment by yogkumgit** [comment]
@efrene

**Comment by yogkumgit** [comment]
@efrene Kindly merge this PR. Thanks

**Comment by tiburd** [comment]
@msbemba 
Hi Sri,
This PR is old, and the procedure has been updated, so I'm closing this PR.
Thanks!

cc: @yogkumgit

**Pull request closed without merging**